etc/ci/check_intermittents.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+REPEAT_COUNT=100
+
+for test_type in wpt css; do
+    while read test_name; do
+        echo "  - Checking $test_name"
+        ./mach test-${test_type} --release --repeat $REPEAT_COUNT "$test_name" > tmp_log.txt
```

Comment:
Let's append to the log file always and have Buildbot handle showing the log (per [this comment](https://github.com/servo/saltfs/pull/453/files#r73552639)). Also, please use the full form and quotes `"${some_variable}"` for the variables (`test_type` and `REPEAT_COUNT`).